doc/audit/ledger.rst

```diff
@@ -35,10 +35,12 @@ The following table describes the structure of a serialised transaction as it is
 +          +------------------------------------------+-------------------------------------------------------------------------+
 |          | uint64_t                                 | Length of serialised public domain                                      |
 +----------+------------------------------------------+-------------------------------------------------------------------------+
-|          | bool                                     | Is snapshot (``false`` for all ledger entries)                          |
+|          | :cpp:type:`kv::EntryType`                | One of WriteSet, Snapshot or WriteSetWithClaims                         |
 +          +------------------------------------------+-------------------------------------------------------------------------+
 |          | :cpp:type:`kv::Version`                  | Transaction version                                                     |
 +          +------------------------------------------+-------------------------------------------------------------------------+
+|          | :cpp:type:`crypto::Sha256Hash`           | User-defined claims digest, only when entry type is WriteSetWithClaims  |
++          +------------------------------------------+-------------------------------------------------------------------------+
 |          | :cpp:type:`kv::Version`                  | Indicates after which version can this ledger entry be executed while   |
 |          |                                          | maintaining linearizability                                             |
```

Comment:
Unrelated to this PR, but is this still true? I believe this was removed in https://github.com/microsoft/CCF/pull/3219

Reply:
We've stopped populating it, but the entry is still there for compatibility.

Reply:
Yes, but the field is only still present to preserve backwards compatibility, so I think we should indicate that here.